addons/web/models/models.py

```diff
@@ -357,3 +357,41 @@ def group_id_name(value):
                 })
 
         return filter_values
+
+
+try:
+    import sass as libsass
+except ImportError:
+    libsass = None
+
+
+class ResCompany(models.Model):
+    _inherit = 'res.company'
+
+    def _get_report_styles(self):
+        template_style = 'web.styles_company_report'
+        if not self or libsass is None or not self.env.ref(template_style, raise_if_not_found=False):
+            return ''
+        styles = []
+        default_color = "#000000"  # black
+        default_layout = 'standard'
+        for company in self:
+            layout_id = company.external_report_layout_id
+            # Provided name are standardized
+            layout = layout_id.name.split('_')[-1] if layout_id else default_layout
```

Comment:
I would remove the "layout_" prefix instead of splitting "_".
And what about using the `.key` instead of the name.

Reply:
Either works for me, doesn't matter much as long as it doesn't crash if someone ever uses a non-standard layout.